.github/workflows/validate-checksums.yml

```diff
@@ -56,5 +56,6 @@ jobs:
               echo
               echo "Please refer to the developer docs at docs/developer for more information."
               echo "-----------------------------------------------------------------------------"
+              ./tools/gen-manifests-diff || true
```

Comment:
I worry this might make the log difficult to read, essentially hiding the instructions, if the diff is large enough.  The message above already guides people towards generating the diff and reviewing it.  I get that it might be useful to have the diff in the log, especially if it's small, but if it's not, it becomes annoying.

Let's keep it for now and see how it goes.

Reply:
I can remove it again, its mostly here for debugging why it currently gives me different results in github than locally, its quite a mystery